content/src/content/jcr_root/apps/core/wcm/components/button/v1/button/README.md

```diff
@@ -0,0 +1,48 @@
+<!--
+Copyright 2019 Adobe Systems Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Button (v1)
+====
+Button component written in HTL.
+
+## Features
+* Linkable to content pages, external URLs or page anchors.
+* Allows an icon identifier to be configured for rendering an icon.
+* Style System support.
+
+### Use Object
+The Button component uses the `com.adobe.cq.wcm.core.components.models.Button` Sling model as its Use-object.
+
+### Edit Dialog Properties
+The following properties are written to JCR for the Button component and are expected to be available as `Resource` properties:
+
+1. `./text` - defines the button text
+2. `./link` - defines the button link
+3. `./icon` - defines an icon identifier for rendering an icon
+
+## BEM Description
+```
+BLOCK cmp-button
+    ELEMENT cmp-button__text
+    ELEMENT cmp-button__icon
+```
+
```

Comment:
I would explain in the README that the icon string is suffixed in the HTL to the CSS class `cmp-button__icon--`.